quick_purchase/models/purchase.py

```diff
@@ -0,0 +1,31 @@
+# coding: utf-8
+# © 2014 Today Akretion
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import ast
+from openerp import api, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    @api.multi
+    def add_product(self):
```

Comment:
Hi @legalsylvain , Thanks for review. 
I thought about this feature, but as I plan to migrate it quickly in version 10.0. This feature is available in this module https://github.com/OCA/purchase-workflow/pull/370/files.